NEWS.md

```diff
@@ -0,0 +1,18 @@
+Version 1.0.1
+-------------
+
+## Unit Testing
+
+  - Unit testing was added in order to have TravisCI checking before any branch merge
+
+## Improvements
+
+  - Added **cache** option to the authentication functions
+  - More readable errors. Also, too much information in the JSON file is now a warning (was an error before)
```

Comment:
This can be a breaking change
Also, ability to read a JSON string and not only a JSON file